src/PKSim.Core/Extensions/FormulaExtensions.cs

```diff
@@ -45,6 +46,20 @@ public static class FormulaExtensions
                objectPath.Replace(keywords[i], replacementValues[i]);
             }
          }
+
+         //---- for sum formulas: replace keywords in tags
+         var sumFormula = formula as SumFormula;
+         if (sumFormula == null)
+            return;
+
+         foreach (var condition in sumFormula.Criteria)
+         {
+            for (int i = 0; i < keywords.Length; i++)
+            {
+               condition.Replace(keywords[i], replacementValues[i]);
+            }
+         }
+
```

Comment:
make sure that keyword extension are applied to the container criteria of sum formulas